docs/stable/14-mentions.md

```diff
@@ -0,0 +1,27 @@
+---
+permalink: /specs/14
+parent: Stable specs
+title: 14/MENTIONS
+---
+
+# 14/MENTIONS
+
+## Usage
+
+To mention a user, one would type a message containing `@0xpk` where `pk` is the public key of the [user account](https://specs.status.im/spec/2) to be mentioned. The message is sent as a [regular message with `TEXT_PLAIN` content type](https://specs.status.im/spec/6#content-types).
+For better user experience, the client should display a known [ens name or the 3-word pseudonym corresponding to the key](https://specs.status.im/spec/2#contact-verification) instead of the `pk`.
```

Comment:
```suggestion
For better user experience, the client SHOULD display a known [ens name or the 3-word pseudonym corresponding to the key](https://specs.status.im/spec/2#contact-verification) instead of the `pk`.
```